pkg/payment/refund/client.go

```diff
@@ -0,0 +1,100 @@
+package refund
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/internal/httpclient"
+)
+
+const (
+	urlBase            = "https://api.mercadopago.com/v1/payments/{id}/"
+	urlRefundWithParam = urlBase + "refunds/{refund_id}"
+	urlRefund          = urlBase + "refunds"
```

Comment:
imo fica melhor assim
```suggestion
	domain            = "https://api.mercadopago.com/v1/payments/{id}"
	urlRefundWithParam = urlBase + "/refunds/{refund_id}"
	urlRefund          = urlBase + "/refunds"
```